WordPress/Classes/ViewRelated/Blog/QuickStartTourGuide.swift

```diff
@@ -16,12 +16,16 @@ open class QuickStartTourGuide: NSObject {
 
     private override init() {}
 
-    func setup(for blog: Blog) {
+    func setup(for blog: Blog, withCompletedSteps steps: [QuickStartTour] = []) {
```

Comment:
What's `[QuickStartTour]` here, is it a generic array of type `QuickStartTour`?

Reply:
Yup, so this is an array of `QuickStartTour` which is a [protocol](https://github.com/wordpress-mobile/WordPress-iOS/blob/259c07e293658b0e57f724f49b26c5651cd10097/WordPress/Classes/ViewRelated/Blog/QuickStartTours.swift#L3-L17) that each of the tour steps conform too. So in the future, if we wanted to add options to complete any of the other steps as part of site creation then we could add their steps to this array as well.